packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -110,6 +112,8 @@ contract DeployConfig is Script {
         sequencerFeeVaultRecipient = stdJson.readAddress(_json, "$.sequencerFeeVaultRecipient");
         sequencerFeeVaultMinimumWithdrawalAmount = stdJson.readUint(_json, "$.sequencerFeeVaultMinimumWithdrawalAmount");
         sequencerFeeVaultWithdrawalNetwork = stdJson.readUint(_json, "$.sequencerFeeVaultWithdrawalNetwork");
+        revenueShareRecipient = payable(stdJson.readAddress(_json, "$.revenueShareRecipient"));
+        revenueShareRemainderRecipient = payable(stdJson.readAddress(_json, "$.revenueShareRemainderRecipient"));
```

Comment:
can you use `_envOr` with null fallback values? This prevents breaking config and also makes it so that the null values are set when the revenue share is turned off